commands.js

```diff
@@ -1490,6 +1491,7 @@ var commands = exports.commands = {
 		this.logModCommand("The ladder was disabled by " + user.name + ".");
 		this.add("|raw|<div class=\"broadcast-red\"><b>Due to high server load, the ladder has been temporarily disabled</b><br />Rated games will no longer update the ladder. It will be back momentarily.</div>");
 	},
+	disableladderhelp: ["Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],
```

Comment:
Why not: "disableladderhelp: ["*/disableladder -* Disables the ladder, which will stop rated battles from being played and stop the ladder from updating. Requires: ~"],"